stable/influxdb/values.yaml

```diff
@@ -271,7 +271,17 @@ backup:
   enabled: false
   schedule: "0 0 * * *"
   annotations: {}
-  destination: gs://bucket/influxdb
-  # Google Cloud Storage
-  gcs:
-    serviceAccountSecret: influxdb-backup-key
+
+  ## Google Cloud Storage
+  # gcs:
+  #   serviceAccountSecret: influxdb-backup-key
+
+  ## Azure
+  ## secret is expected to have `connection-string` field
+  # azure:
+  #   storageAccountSecret: influxdb-backup-connection-string
+
+  ## For Azure
+  # destination: influxdb-container
```

Comment:
I believe there may need consistency, that we don't need to maintain the same field just for different cloud provider.

Reply:
Should we make this to below?

```
gcs:
  serviceAccountSecret: influxdb-backup-ke
  destination: gs://bucket/influxdb
```

```
azure:
  storageAccountSecret: influxdb-backup-connection-string
  destination: influxdb-container
```

Reply:
OK, but let me make sure: can we drop backward-compatibility?

Reply:
Yes, the backup-feature is needed more simple to use. There may make some major breaking. And I'd like to follow @maorfr solution, What did you think?

Reply:
Ok, so let's drop backward-compatibility :+1: 

Regarding the @maorfr solution, I have mixed feelings:
- I will be nice to have universal tool that supports both GCP, AWS, Azure, so it's a plus for `skbn`.
But:
- I would prefer to depend on big-brand official images than small, personal libraries.
- As far as I understand how this `skbn` tool works, it seems to me that it is designed to copy existing files from pods. Here, we have no files to be simply uploaded, so anyway, we need to create temporary volume where `influxd` can store the output of the backup. So one of advantages of `skbn` is lost.
- Using that solution would require actually dropping most of the changes I proposed here, I am not sure I have now enough capacity for rewriting it from the scratch.

Reply:
I think we can merge this first for support Azure. There may have another PR to reinstruct the backup feature. And  I agree with you `depend on big-brand official images`. And I need more time to research `skbn`.

Reply:
OK, great.

I should also update the values to the format you proposed, right?

Reply:
Yes, bump the minor version to `1.4.0` of this chart.

Reply:
> Should we make this to below?
> 
> ```
> gcs:
>   serviceAccountSecret: influxdb-backup-ke
>   destination: gs://bucket/influxdb
> ```
> 
> ```
> azure:
>   storageAccountSecret: influxdb-backup-connection-string
>   destination: influxdb-container
> ```

And shall I use this as well or not?

Reply:
Yes, I believe this can be useful.